djangoevents/tests/test_domain.py

```diff
@@ -25,30 +26,46 @@ def __init__(self, is_dirty=False, **kwargs):
 
 
 def test_base_entity_calls_mutator():
-    create_event = SampleEvent.Created(entity_id=1)
-    entity = SampleEvent.mutate(event=create_event)
+    create_event = SampleEntity.Created(entity_id=1)
+    entity = SampleEntity.mutate(event=create_event)
 
-    close_event = SampleEvent.Updated(entity_id=entity.id, entity_version=entity.version)
-    entity = SampleEvent.mutate(event=close_event, entity=entity)
+    close_event = SampleEntity.Updated(entity_id=entity.id, entity_version=entity.version)
+    entity = SampleEntity.mutate(event=close_event, entity=entity)
     assert entity.is_dirty is True
 
+
 def test_base_entity_requires_mutator():
-    create_event = SampleEvent.Created(entity_id=1)
-    entity = SampleEvent.mutate(event=create_event)
+    create_event = SampleEntity.Created(entity_id=1)
+    entity = SampleEntity.mutate(event=create_event)
 
-    close_event = SampleEvent.Closed(entity_id=entity.id, entity_version=entity.version)
+    close_event = SampleEntity.Closed(entity_id=entity.id, entity_version=entity.version)
 
     with pytest.raises(NotImplementedError):
-        SampleEvent.mutate(event=close_event, entity=entity)
+        SampleEntity.mutate(event=close_event, entity=entity)
 
 
 def test_create_for_event():
-    event = SampleEvent.Created(
+    event = SampleEntity.Created(
         entity_id='ENTITY_ID',
         domain_event_id='DOMAIN_EVENT_ID',
         entity_version=0,
     )
-    obj = SampleEvent.create_for_event(event)
+    obj = SampleEntity.create_for_event(event)
 
     assert obj.id == 'ENTITY_ID'
     assert obj.version == 0
+
+
+def test_list_aggregates():
+    aggregates = list_concrete_aggregates()
+    assert len(aggregates) == 1
+    assert aggregates[0] is SampleEntity
+
+
+def test_list_events_sample_event():
+    events = list_aggregate_events(SampleEntity)
+    assert len(events) == 2
```

Comment:
next assertion already checks if there are 2 events. And having `len` based check first makes it impossible to see differences in case something would be different here. Let's just leave `set` based check.